frontend/src/Editor/Inspector/Inspector.jsx

```diff
@@ -182,71 +186,50 @@ export const Inspector = ({
-            />
-            <span className="input-icon-addon">
-              <img src="/assets/images/icons/edit-source.svg" width="12" height="12" />
-            </span>
-          </div>
-        </div>
-        <div className="col py-1">
-          <button className="btn btn-sm component-action-button btn-light" onClick={() => switchSidebarTab(2)}>
-            x
-          </button>
-        </div>
-      </div>
-
-      {componentMeta.component === 'Table' && (
-        <Table
-          component={component}
-          paramUpdated={paramUpdated}
-          dataQueries={dataQueries}
-          componentMeta={componentMeta}
-          eventUpdated={eventUpdated}
-          eventOptionUpdated={eventOptionUpdated}
-          components={components}
-          currentState={currentState}
-          darkMode={darkMode}
-          eventsChanged={eventsChanged}
-          apps={apps}
-        />
-      )}
-
-      {componentMeta.component === 'Chart' && (
-        <Chart
-          component={component}
-          paramUpdated={paramUpdated}
-          dataQueries={dataQueries}
-          componentMeta={componentMeta}
-          eventUpdated={eventUpdated}
-          eventOptionUpdated={eventOptionUpdated}
-          components={components}
-          currentState={currentState}
-          darkMode={darkMode}
-        />
-      )}
-
-      {!['Table', 'Chart'].includes(componentMeta.component) && (
-        <div className="properties-container p-2">
-          {Object.keys(componentMeta.properties).map((property) =>
+  function getAccordion(componentName) {
+    switch (componentName) {
+      case 'Table':
+        return (
+          <Table
+            layoutPropertyChanged={layoutPropertyChanged}
+            component={component}
+            paramUpdated={paramUpdated}
+            dataQueries={dataQueries}
+            componentMeta={componentMeta}
+            eventUpdated={eventUpdated}
+            eventOptionUpdated={eventOptionUpdated}
+            components={components}
+            currentState={currentState}
+            darkMode={darkMode}
+            eventsChanged={eventsChanged}
+            apps={apps}
+          />
+        );
+
+      case 'Chart':
+        return (
+          <Chart
+            layoutPropertyChanged={layoutPropertyChanged}
+            component={component}
+            paramUpdated={paramUpdated}
+            dataQueries={dataQueries}
+            componentMeta={componentMeta}
+            eventUpdated={eventUpdated}
+            eventOptionUpdated={eventOptionUpdated}
+            components={components}
+            currentState={currentState}
+            darkMode={darkMode}
+          />
+        );
+
+      default: {
+        const properties = Object.keys(componentMeta.properties);
+        const events = Object.keys(componentMeta.events);
+        const validations = Object.keys(componentMeta.validation || {});
+        let items = [];
+        items.push({
+          title: 'Basic fields',
```

Comment:
Can the title be "Properties" instead of "Basic fields"?

Reply:
Ok sure made the change 👍